sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/ConfidentialLedgerClientLiveTests.cs

```diff
@@ -39,6 +39,14 @@ public ConfidentialLedgerClientLiveTests(bool isAsync) : base(isAsync)
         {
             // https://github.com/Azure/autorest.csharp/issues/1214
             TestDiagnostics = false;
+            // Add BodyRegexSanitizer to handle newline characters
+            BodyRegexSanitizers.Add(
+                new BodyRegexSanitizer(
```

Comment:
This isn't being correctly defined. Please look at the sample I linked in the Teams channel.